lib/react_client.dart

```diff
@@ -162,11 +162,9 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
 
   /// Wrapper for [Component.getInitialState].
   void initComponent(ReactComponent jsThis, ReactDartComponentInternal internal, ComponentStatics componentStatics) => zone.run(() {
-    var redraw = () {
-      if (internal.isMounted) {
```

Comment:
This check always evaluated to true after mounting since this `internal` was never updated, so this can be safely removed.